sourced/ml/mining/spiders/go_stdlib.py

```diff
@@ -0,0 +1,23 @@
+import scrapy
+
+
+class GoStdlibSpider(scrapy.Spider):
+    """Spider for scraping the Go standard libraries."""
+
+    name = "go-stdlib-spider"
+
+    def start_requests(self):
+        """Start making requests."""
+        for url, kind in zip(
+            ["https://godoc.org/-/go", "https://godoc.org/-/subrepo"],
```

Comment:
This bends my mind less:
```suggestion
            for url, kind in [("https://godoc.org/-/go", "core"),
                              ("https://godoc.org/-/subrepo", "subrepo")],
```

Reply:
done :p